op-challenger/scripts/alphabet/init_game.sh

```diff
@@ -34,10 +34,10 @@ echo " Block Oracle Proxy at $BLOCK_ORACLE_PROXY"
 echo "----------------------------------------------------------------"
 
 CHARLIE_ADDRESS="0xF45B7537828CB2fffBC69996B054c2Aaf36DC778"
-CHARLIE_KEY="74feb147d72bfae943e6b4e483410933d9e447d5dc47d52432dcc2c1454dabb7"
+# CHARLIE_KEY="74feb147d72bfae943e6b4e483410933d9e447d5dc47d52432dcc2c1454dabb7"
```

Comment:
Deleting these unused keys would be fine. They're set in the other scripts that actually use them.

Reply:
Deleted in 591d885f7